src/index.js

```diff
@@ -58,28 +61,4 @@ Object.values(components).forEach((component) => {
   }
 });
 
-export {
-  Input,
-  DateInput,
-  TextInput,
-  Button,
-  RadioButtons,
-  PlainButton,
-  ButtonBar,
-  NavBar,
-  Modal,
-  Details,
-  Card,
-  CheckBox,
-  Dropdown,
-  Tooltip,
-  LoadingShim,
-  LoadingSkeleton,
-  Dialog,
-  Row,
-  Pagination,
-  SeparatorList,
-  Popover,
-  Toggle,
-  Tabs,
-};
+module.exports = Object.assign({}, components, formatters);
```

Comment:
This big list of exports is already stored in `const components` on L28.
Here I am merging formatters into the exports object so they can be imported in consumers like this:

`import { formatNumber } from '@narmi/design_system';`